modules/platforms/cpp/thin-client-test/include/vector_logger.h

```diff
@@ -67,7 +67,7 @@ namespace ignite_test
             const std::string& nativeErrorInfo)
         {
             if ((categoryFilter.empty() || category.find(categoryFilter) != std::string::npos) &&
-                (messageFilter.empty() || message.find(messageFilter) != std::string::npos))
+                (!messageFilter.empty() && message.find(messageFilter) != std::string::npos))
```

Comment:
Can you please explain the change? Why the second condition is different from the first one?